app/pages/household/[houseId]/utility/[utilityId]/index.tsx

```diff
@@ -95,6 +97,14 @@ const Utility = () => {
     }
   };
 
+  const Router = useRouter();
+
+  const handleDeleteUtility = async (event: any) => {
+    event.preventDefault();
+    Router.push("/");
+    await deleteDoc(amenityRef);
+  };
+
```

Comment:
Perhaps go to household page after deleting utility (/[houseId]) instead of homepage?